### PR TITLE
Make: Landscape the default headlines layout

### PR DESCRIPTION
**Pull request by andygout**
@ironsidevsquincy @musthemag 

Following the negative reaction to the seldom-seen `Standard` layout last week, we are removing that option from the headlines layouts and making the default the landscape layout.

- [x] Approved by Editorial.
- [x] 'Standard' option removed from Methode CMS.
- [x] 'Standard' option removed from Next Flags API (and by association the Feature Toggle).

### CODE REVIEW

File: config/sections/headlines.js [hunk 1]
```diff
@@ -9,9 +9,8 @@ const getLayout = (layoutHint) => {
 		case 'assassination':
 		case 'bigstory':
 			return 'headlines-big-story';
-		case 'standard':
 		default:
-			return 'headlines';
+			return 'headlines-landscape';
```
Comment (ironsidevsquincy) [inline config/sections/headlines.js]: can remove the `landscape` case from this, right?

File: server/libs/section-data.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const headlineStoryCount = layout => {
 		case 'bigstory': return 4;
 		case 'landscape': return 1;
 		case 'standaloneimage': return 2;
-		default: return 5;
+		default: return 1;
```
Comment (ironsidevsquincy) [inline server/libs/section-data.js]: ditto

**Comment by ironsidevsquincy** [comment]
Will need to update tests

File: config/sections/headlines.js (hunk 1 above)
Comment (andygout) [inline config/sections/headlines.js]: Reckon it's better to have `landscape` share with `default` so it's more explicit what is being returned when the layout hint is set to that option (it did this previously with `standard`).

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**